### PR TITLE
feat: add LLM api

### LINKED ISSUE #372: able to manage LLMs in apiserver

### PR DESCRIPTION
**Pull request by Lanture1064**
#### What type of PR is this?

/kind feature

#### Which issue(s) this PR fixes

Fixes #372

### CODE REVIEW

File: graphql-server/go-server/pkg/llm/llm.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		labelSelector = *input.LabelSelector
+	}
+	if input.Page != nil && *input.Page > 0 {
+		page = *input.Page
+	}
+	if input.PageSize != nil && *input.PageSize > 0 {
+		pageSize = *input.PageSize
+	}
+
+	dsSchema := schema.GroupVersionResource{Group: v1alpha1.GroupVersion.Group, Version: v1alpha1.GroupVersion.Version, Resource: "embedders"}
```
Comment (Abirdcfly) [inline graphql-server/go-server/pkg/llm/llm.go:91]: embedders -> llms?

File: graphql-server/go-server/graph/impl/llm.resolvers.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import (
+	"context"
+	"fmt"
+
+	"github.com/kubeagi/arcadia/graphql-server/go-server/graph/generated"
+)
+
+// GetLlm is the resolver for the getLLM field.
+func (r *lLMQueryResolver) GetLlm(ctx context.Context, obj *generated.LLMQuery, name string, namespace string) (*generated.Llm, error) {
+	panic(fmt.Errorf("not implemented: GetLlm - getLLM"))
```
Comment (Abirdcfly) [inline graphql-server/go-server/graph/impl/llm.resolvers.go:16]: maybe need some code.

File: graphql-server/go-server/graph/impl/llm.resolvers.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+// ListLLMs is the resolver for the listLLMs field.
+func (r *lLMQueryResolver) ListLLMs(ctx context.Context, obj *generated.LLMQuery, input generated.ListCommonInput) (*generated.PaginatedResult, error) {
+	panic(fmt.Errorf("not implemented: ListLLMs - listLLMs"))
```
Comment (Abirdcfly) [inline graphql-server/go-server/graph/impl/llm.resolvers.go:21]: same

File: graphql-server/go-server/graph/impl/llm.resolvers.go [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+// Llm is the resolver for the LLM field.
+func (r *queryResolver) Llm(ctx context.Context) (*generated.LLMQuery, error) {
+	panic(fmt.Errorf("not implemented: Llm - LLM"))
```
Comment (Abirdcfly) [inline graphql-server/go-server/graph/impl/llm.resolvers.go:26]: same

File: graphql-server/go-server/graph/impl/llm.resolvers.go (hunk 2 above)
Comment (bjwswang) [reply graphql-server/go-server/graph/impl/llm.resolvers.go:16]: @Lanture1064

File: graphql-server/go-server/pkg/llm/llm.go [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+	return &generated.PaginatedResult{
+		TotalCount:  totalCount,
+		HasNextPage: end < totalCount,
+		Nodes:       result,
+	}, nil
+}
+
+// ReadLLM
+func ReadLLM(ctx context.Context, c dynamic.Interface, name, namespace string) (*generated.Llm, error) {
+	resource := c.Resource(schema.GroupVersionResource{Group: v1alpha1.GroupVersion.Group, Version: v1alpha1.GroupVersion.Version, Resource: "llms"})
```
Comment (bjwswang) [inline graphql-server/go-server/pkg/llm/llm.go:143]: Please use schemas in https://github.com/kubeagi/arcadia/blob/main/graphql-server/go-server/pkg/common/schema.go#L66

**Pull request merged**